gui/default/index.html

```diff
@@ -116,14 +116,18 @@ <h3 class="panel-title">
               <span class="caret"></span>
             </a>
             <ul class="dropdown-menu">
```

Comment:
Currently this leaves us with an empty "Actions" menu on the login page; should the entire menu be removed since it's empty?

Reply:
I've no opinion either way. Shall I go ahead and make that change?

Reply:
Definitely, please hide the empty menu.

Reply:
Done in 1e87c9aeb00d7215dd7f166985ec7553268c23ce.